app/models/reservation.rb

```diff
@@ -147,12 +151,15 @@ def end_at_required?
 
   # Is there enough information to move an associated order to complete/problem?
   def order_completable?
-    actual_end_at || reserve_end_at < Time.current
+    force_completable || actual_end_at || reserve_end_at < Time.current
   end
 
   def start_reservation!
-    # Mark running reservations as complete, which will move them to problem orders
-    product.schedule.products.flat_map(&:started_reservations).each(&:complete!)
+    # If there are any reservations running over their time on the shared schedule,
+    # kick them over to the problem queue.
+    product.schedule.products.flat_map(&:started_reservations).each do |reservation|
+      MoveToProblemQueue.move!(reservation.order_detail)
```

Comment:
Should we set `force_complete` here as well, to ensure that we can kick them over?

Reply:
Probably a good idea.

Reply:
Hmm, the test failure appears related to this seems legit. I'm thinking this one should not be `force_complete`. I'll need to dig into it a little more.

"ReservationsController needs now reservation switch_instrument on before the reservation start (in grace period) in the grace period, but there is another reservation still running does not start the reservation - spec.controllers.reservations_controller_spec"